tests/e2e/scaling_test.go

```diff
@@ -28,11 +28,13 @@ import (
 
 var _ = Describe("Cluster scale up and down", func() {
 	const (
-		namespace   = "cluster-scale-e2e-storage-class"
-		sampleFile  = fixturesDir + "/base/cluster-storage-class.yaml.template"
-		clusterName = "postgresql-storage-class"
-		level       = tests.Lowest
+		namespace                 = "cluster-scale-e2e-storage-class"
+		sampleFileWithoutRepSlots = fixturesDir + "/base/cluster-storage-class.yaml.template"
+		sampleFileWithRepSlots    = fixturesDir + "/base/cluster-storage-class-with-rep-slots.yaml.template"
+		clusterName               = "postgresql-storage-class"
+		level                     = tests.Lowest
 	)
+	sampleFile := sampleFileWithRepSlots
```

Comment:
yikes. We're changing this test, as well as the switchover / failover tests.
And no longer test without replication slots (except in postgres 10).
We should add new tests, not subtract!!